externs/v20140218_eme.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright 2014 Google Inc.
```

Comment:
2015
Also, add a license annotation just before the copyright header, as in 4cc4e96?